src/SongList.tsx

```diff
@@ -0,0 +1,6 @@
+import React, { useState } from "react";
+import { useHandleFetchAndLoad } from "./useHandleFetchAndLoad";
+
+export const Songlist = (props: {playlistId: object}): React.ReactElement => {
```

Comment:
is there a reason this is an extension and not a class? not familiar with react specifically with typescript but this seems very pre-es6

Reply:
discard this